server/services/interventionsService.ts

```diff
@@ -95,6 +105,34 @@ export default class InterventionsService {
     return sanitisedError
   }
 
+  serializeDeliusServiceUser(deliusServiceUser: DeliusServiceUser | null): ServiceUser {
+    if (!deliusServiceUser) {
+      return {} as ServiceUser
+    }
+
+    const currentDisabilities = deliusServiceUser.disabilities
+      .filter(disability => {
+        const today = new Date().toString()
+        return disability.endDate === '' || Date.parse(disability.endDate) >= Date.parse(today)
+      })
+      .map(disability => disability.disabilityType.description)
+
+    const iso8601DateOfBirth = CalendarDay.parseIso8601(deliusServiceUser.dateOfBirth)?.iso8601 || null
```

Comment:
Why are we parsing an ISO 8601 string and then converting the result back to ISO 8601? Is it to validate that it's correctly formatted?

Reply:
Yeah, I wasn't sure if we could rely on it being in the correct format, but maybe we can...

Reply:
I think it's sensible to check. Our ultimate goal is to be checking everything, so why not start here!